lib/excon/constants.rb

```diff
@@ -119,10 +119,10 @@ module WaitWritable; end
     :idempotent           => false,
     :instrumentor_name    => 'excon',
     :middlewares          => [
+      Excon::Middleware::Instrumentor,
```

Comment:
having it above `Idempotent` means you don't log the retry requests, but having it below `Idempotent` means you don't have access to the parsed response.